### DISCUSSION TITLE
Should we use copy or reference of Uuid value?

### DISCUSSION THREAD

**Discussion by kvzn** [uuid-rs/uuid#627] [Q&A] [answered]
Uuid is copyable, but should wen use copy OR reference when we are passing it? Thank you!

**Comment by KodrAus** [comment, marked as answer]
Hi @kvzn :wave:

You can typically just pass `Uuid`s around by-value, but sometimes it's more efficient to pass them by-reference. As a general rule though, I'd pass them by-value and then decide if by-reference is appropriate on a case-by-case basis.
